README.rst

```diff
@@ -421,10 +421,10 @@ please join the `celery-users`_ mailing list.
 IRC
 ---
 
-Come chat with us on IRC. The **#celery** channel is located at the `Freenode`_
-network.
+Come chat with us on IRC. The **#celery** channel is located at the
+`Libera Chat`_ network.
 
-.. _`Freenode`: https://freenode.net
+.. _`Libera Chat`: https:/libera.chat/
```

Comment:
```suggestion
.. _`Libera Chat`: https://libera.chat/
```

Reply:
Thanks for noticing that goof on my part!

Reply:
I had to look into this, aparantly any number (one or more) of slashes is acceptable